### PR TITLE
feat(switch): add Switch component

### LINKED ISSUE #69: Switch
## 🚀 Feature Proposal

Add a Switch component to the library.

## Motivation

Switches are a simple UI technique to turn something on or off. 

## Example

Make an appointment an all day appointment.

### PR DESCRIPTION
**Pull request by jackcmeyer**
Fixes #69

**Changes proposed in this pull request:**
- Adds Switch component

### CODE REVIEW

**Comment by tehKapa** [comment]
@jackcmeyer like Travis, I can't build on my Mac. 

`(typescript) Error: src/components/Switch/Switch.tsx:25:11 - error TS2769: No overload matches this call.`

File: src/components/Switch/Switch.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+import FormCheck from 'react-bootstrap/FormCheck'
+
+interface Props {
+  /** The id for the switch element */
+  id: string
+  /** The label to render next to the switch */
+  label: string
+  /**Gives the switch a name */
+  name?: string
```
Comment (irvelervel) [inline src/components/Switch/Switch.tsx]: Is this property useful in the FormCheck component? I can't find it in the documentation

**Comment by irvelervel** [comment]
> We need to test the deps _@fortawesome/free-solid-svg-icons_. It's not downloadable on Windows using yarn
> 
> UPDATE: now it seems okay

Yes, this problem seems to happen from time to time...

File: src/components/Switch/Switch.tsx (hunk 1 above)
Comment (irvelervel) [reply src/components/Switch/Switch.tsx]: @jackcmeyer

Comment (jackcmeyer) [reply src/components/Switch/Switch.tsx]: Hmmm... I think it probably can be removed

Comment (fox1t) [reply src/components/Switch/Switch.tsx]: Are we going to remove this?

Comment (fox1t) [reply src/components/Switch/Switch.tsx]: @irvelervel ?

Comment (jackcmeyer) [reply src/components/Switch/Switch.tsx]: @fox1t @irvelervel I have removed it :)

Comment (irvelervel) [reply src/components/Switch/Switch.tsx]: 👍

**Pull request merged**